plugins/bed/src/BigBedAdapter/BigBedAdapter.ts

```diff
@@ -113,6 +113,9 @@ export default class BigBedAdapter extends BaseFeatureDataAdapter {
               if (r.uniqueId === undefined) {
                 throw new Error('invalid bbi feature')
               }
+              delete data.chromStart
+              delete data.chromEnd
+              delete data.chrom
 
```

Comment:
if we do this for bigbed, we should probably do it for bedtabix as well, don't you think?

Reply:
pushed this change onto master so as not to muddle this PR, but bedtabix already had this